.python-version

```diff
@@ -1 +1 @@
-3.12
+3.11
```

Comment:
Should that not be a separate PR, if it is needed?

Reply:
The pre-commit didn't work with Python 3.12 -> not related to this, but needed :-)
OK => separate pull request :-)